common/util/file_util.cc

```diff
@@ -290,6 +290,9 @@ absl::StatusOr<Directory> ListDir(absl::string_view dir) {
   return d;
 }
 
+static const char *STDIN_FILENAME = "-";
```

Comment:
To limit the scope, I suggest to put this constant in the `IsStdin()` function.

Constants should have a `k` prefix and otherwise be camel-case according [to the styleguide we use](https://google.github.io/styleguide/cppguide.html#Constant_Names).

Also, if you make it an `absl::string_view` we save the strlen() call at runtime.

```
bool IsStdin(absl::string_view filename) {
  static constexpr absl::string_view kStdinFilename = "-";
  return filename == kStdinFilename;
}
```

Reply:
Thanks for the feedback! Sorry about the naming convention

Reply:
I've fixed the smoke-test expecations in #1849 so if you do a rebase, that should work then.

Reply:
Tried to avoid having a merge commit with my own branch and ended up making a mess. I'll try going back and just adding the merge commit.